src/address.rs

```diff
@@ -325,47 +329,3 @@ impl<A, Rc: RefCounter> Hash for Address<A, Rc> {
         state.finish();
     }
 }
-
-/// A sink which can be used to send messages to an actor. These messages will have default priority
-/// and be handled in send order.
-pub struct AddressSink<A, Rc: RefCounter>(inbox::SendFuture<A, Rc>);
-
-impl<A, Rc: RefCounter> AddressSink<A, Rc> {
-    /// Return a clone of the underlying [`Address`] which this sink sends to.
-    pub fn address(&self) -> Address<A, Rc> {
```

Comment:
The main thing if we implement this change is losing this method (and the nameability of the type)

Reply:
Also, Rust will hopefully soon support impl-trait-type-aliases! :)